types/node/test/events.ts

```diff
@@ -71,10 +71,6 @@ const any: any = 1;
     }
 }
 
-{
-    new events();
```

Comment:
actually this should work, see https://github.com/nodejs/node/blob/64eacd8741f23cb91569faf7ec793c83badb04e3/lib/events.js#L46

Reply:
Interesting, have you seen this used anywhere in the ecosystem?
Things like these are super annoying to model and usually require hacks such as fake inheritance.
Any ideas?

Reply:
Can't remember a specific place; I expect this is a relict from node 0.x so if this is used somewhere then most likely in old modules.
It's the same/similar for `module` (see https://github.com/nodejs/node/blob/v13.1.0/lib/internal/modules/cjs/loader.js#L1261) and `stream` as also similar.
What was the problem with the exisiting variant `export = internal`?

Reply:
I figured v13 was a good opportunity to clean things up a bit and remove this `virtual` inheritance nonsense, but it sounds like this could break a few people 🤔

Reply:
I'm actually pretty curious to see how many people would be affected by this.
Do you think it would be an acceptable risk to do this on v13 or does it have too much potential for breaking changes?

Reply:
As far as I know ES6 modules don't allow such exports anyway so seems to be a step forward. Fixing an app is easy so I'm ok with it. Reverting in case of a lot complains should be also not that hard.

Reply:
Same applies to `module`, `stream`. Although I won't be touching `stream` for now.

Reply:
Honestly speaking I would be also careful with `module` as this is usually just required to get the static methods exposed there. So at least these should be kept.